index.html

```diff
@@ -0,0 +1,201 @@
+        <p>In exchange, Lambda School graduates pay back a small percentage of their salary after they are hired.</p>
+      </div>
+      <div class="TextColumn">
+        <h5 style="color:black;font-weight:bold";>Selection</h5>
+        <p>Applicants run through a battery of tests and interviews designed to identify those with the greatest aptitude for a highly technical career.</p>
+      </div>
+      <div class="TextColumn">
+        <h5 style="color:black;font-weight:bold";>Training</h5>
+        <p>Once students are selected, a small class undergoes live, intensive online training designed to meet the rigorous demands of our hiring partners.</p>
+      </div>
+    </div>
+  </div>
+  <div class="OffWhiteSection" id="Courses">
+    <h3 style="color:black;font-weight:bold";>Lambda School Courses</h3>
+    <div class="ContainerCourses">
+      <div class="WhiteRectangle">
+        <h5 style="color:black;font-weight:bolder";>COMPUTER SCIENCE ACADEMY</h5>
+        <p>6 months (full-time)</p>
+        <p>Web &amp Software Engineering</p>
+        <p>$0 up-front</p>
+        <p>+17% of salary for 2 yrs</p>
+        <p>(or $20,000 up-front)</p>
+        <button type="button" class="BlueButton">Learn More</button>
+      </div>
+      <div class="card1">
+        <h5 style="color:black;font-weight:bolder";>MACHINE LEARNING &amp AI ACADEMY</h5>
+        <p>6 months (full-time)</p>
+        <p>Machine &amp Deep Learning</p>
+        <p>$0 up-front</p>
+        <p>+17% of salary for 2 yrs</p>
+        <p>(or $20,000 up-front)</p>
+        <button type="button" class="BlueButton">Learn More</button>
+      </div>
+      <div class="card1">
+        <h5 style="color:black;font-weight:bolder;";>WEB DEVELOPMENT BOOTCAMP</h5>
+        <p>13-17 weeks (full-time/part-time)</p>
+        <p>Full-Stack Web Development</p>
+        <p>$5,000 part-time</p>
+        <p>$10,000 full-time</p>
+        <p>financing available</p>
+        <button type="button" class="BlueButton">Learn More</button>
+      </div>
+    </div>
+  </div>
+  <div class="ContainerSkills">
+    <div class="BluePicture">
+      <img src="https://lambdaschool.com/assets/img/istock1blue.jpg" alt="Blue Picture">
+      <p>Picture here</p>
+    </div>
+    <div class="WhiteSection" id="Skills">
+        <h3>The skills you need, wherever you are</h3>
+        <p>Lambda School is streamed live from Silicon Valley to a small, interactive class located all over the world.</p>
+        <p>Featuring world-class instructors, cutting edge curriculum, and plenty of one-on-one help, you can attend Lambda School from wherever you happen to be.</p>
+    </div>
+  </div>
+  <div class="WhiteSection" id="RiskFree">
+    <div class="container">
+      <div class="WhiteSection" id="RiskSpan">
+        <h3 style="color:black";>Graduate Risk-Free</h3>
+        <p>Only begin paying tuition after you have a job.</p>
+        <p>Only continue paying while  you're making more than $50,000/year.</p>
+	<br>
```

Comment:
I think that the `<br>` tag is a special case, but most self closing elements should be self closed `<br/>`